docs/discover/search.asciidoc

```diff
@@ -2,10 +2,10 @@
 == Searching Your Data
 You can search the indices that match the current index pattern by entering
 your search criteria in the Query bar. You can use Kibana's standard query language
-(based on Lucene https://lucene.apache.org/core/2_9_4/queryparsersyntax.html[query syntax]) 
-or the full JSON-based {ref}/query-dsl.html[Elasticsearch Query DSL]. Autocomplete 
-and a simplified query syntax are available for the Kibana query language as experimental
-features which you can opt-in to under the options menu in the Query Bar. 
+which features autocomplete and a simple, easy to use syntax or you can use the
+full JSON-based {ref}/query-dsl.html[Elasticsearch Query DSL]. Kibana's legacy query
```

Comment:
I don't think we support this when KQL is enabled...

Reply:
Good point, I've pushed a change to make this more clear.